src/api/products.ts

```diff
@@ -1,15 +1,22 @@
 import { ctpClient } from './commerceToolsClient';
+import type { ApiRoot, ClientResponse, ProductPagedQueryResponse } from '@commercetools/platform-sdk';
 import { createApiBuilderFromCtpClient } from '@commercetools/platform-sdk';
+import { AuthData as AUTH } from '@/api/token/authData';
 
-const apiRoot = createApiBuilderFromCtpClient(ctpClient);
+const apiRoot: ApiRoot = createApiBuilderFromCtpClient(ctpClient);
 
-export const fetchAllProducts = async () => {
-  try {
-    const response = await apiRoot.withProjectKey({ projectKey: 'awesome-coffee-shop' }).products().get().execute();
+export const fetchAllProducts: () => Promise<ProductPagedQueryResponse> =
+  async (): Promise<ProductPagedQueryResponse> => {
```

Comment:
```suggestion
export const fetchAllProducts =
  async (): Promise<ProductPagedQueryResponse> => {
```
we don't need to define type two times, it would be enough above one